src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -47,13 +48,12 @@ const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
             </div>
             <div className="col-lg-6">
               <img src={datastudiogif} width="800" height="500" />
```

Comment:
Sería bueno que cuando se agreguen las imagenes, colocar siempre un atributo alt con un texto alternativo coherente a la imagen. Esta info es necesaria para que el buscador la identifique con un contenido concreto que pueda catalogar e indexar

Reply:
suggest accepted and applied